### PR TITLE
Update plugins.xml

### PR DESCRIPTION
**Pull request by ronste**
Update Shariff plugin with WCAG compliant buttons styles and optional block heading.

### CODE REVIEW

**Comment by asmecher** [comment]
@ronste, it looks like this PR modifies an existing release rather than adding a new one; can you change it to add a new release stanza?

**Comment by ronste** [comment]
> @ronste, it looks like this PR modifies an existing release rather than adding a new one; can you change it to add a new release stanza?

Yes, that was my intention. I inserted the previous releases again.

**Comment by ronste** [comment]
> @ronste, it looks like the MD5s didn't match. Can you double-check?

Hi @asmecher , that's strange. I can't find any issue there:

shariff-3_3_0-1.tar.gz => 970b2ae418dec259c2e84e4364cc79cb

shariff-3_3_0-2.tar.gz => de3b2530fe5aa1737ae62517e9fe5416

shariff-3_4_0.tar.gz => 319d33932cc130e592768e6d9575f63a

shariff-3_4_0-1.tar.gz => 61db66ffb5b4398763d184165ae8a725

Any other options what might have went wrong?

**Comment by asmecher** [comment]
@ronste, I'll try the build again and see if it was a network hiccough or something.

File: plugins.xml [hunk 1]
```diff
@@ -2198,6 +2198,43 @@
 			</compatibility>
 			<certification type="partner"/><description>This plugin adds social media buttons to your web site without compromising the privacy of website users.</description>
 		</release>
+		<release date="2023-09-18" version="3.3.1.0" md5="de3b2530fe5aa1737ae62517e9fe5416">
```
Comment (asmecher) [inline plugins.xml:2201]: @ronste, is the version number wrong here? The archive filename says `3_3_0-2`, and the `version` attribute says `3.3.1.0`.

Comment (ronste) [reply plugins.xml:2201]: > @ronste, is the version number wrong here? The archive filename says `3_3_0-2`, and the `version` attribute says `3.3.1.0`.

@asmecher , no, I can't see any issues there either. The version numbers seem to be ok. Historically the OJS version numbers have been used as release tags in this repo. Now that I have a second release within a major OJS version I also had to start numering. So filenames, versioning and tags now are:

shariff-3_3_0-1.tar.gz => plugin version 3.3.0.1 => 970b2ae418dec259c2e84e4364cc79cb

shariff-3_3_0-2.tar.gz => plugin version 3.3.1.0 => de3b2530fe5aa1737ae62517e9fe5416

shariff-3_4_0.tar.gz => plugin version 3.4.0.0 => 319d33932cc130e592768e6d9575f63a

shariff-3_4_0-1.tar.gz => plugin version 3.4.2.0 => 61db66ffb5b4398763d184165ae8a725

That's weird, I can't see any issue. There's a tabulator in excess but that's all.

Comment (asmecher) [reply plugins.xml:2201]: Ah, I think I was being thrown off by the version numbers, but that's a red herring. The plugin gallery toolset is only expecting one added plugin per pull request, and only gets the first MD5. I'll merge this as is and let it test fully, but next time, I'd suggest just adding one plugin release per PR.

Comment (ronste) [reply plugins.xml:2201]: Ahh, I see. I'll try to remember this for the next PR. Thanks!

**Pull request merged**